syft/licensectx/context.go

```diff
@@ -0,0 +1,13 @@
+package licensectx
```

Comment:
We have some exported license-related things here: https://github.com/anchore/syft/blob/main/syft/cataloging/license.go, maybe instead of making a new package this would be a better spot to include the exported context key and creation function? These seem like configuration items, even though they are using context to provide them.

Reply:
Thanks for the suggestion -  that's a good place for these! Have moved them :)